src/util/integrationIconPath.ts

```diff
@@ -0,0 +1,8 @@
+const ICON_BASE_URL = 'https://raw.githubusercontent.com/JupiterOne/docs/master/assets/icons/';
+
+export default function integrationIconPath(integrationType: string): string {
+  if (integrationType === 'cbdefense') {
+    return `${ICON_BASE_URL}carbon-black-psc.svg`;
+  } 
+  return `${ICON_BASE_URL}${integrationType.replace(/\s|_/g, '-')}.svg`;
```

Comment:
I have to admit, I haven't following along super close, but looking at https://apps.us.jupiterone.io/static/managed-questions.json, where I believe that `managedQuestions.integrations[question.integration].type` is coming from (the argument to this function), I don't believe there will ever be a space in the `type`, since it is the vendor ID. I think it would be ideal to have images always use the name of the integration definition vendor ID, or we'll end up with code in various places attempting to map the vendor ID (or worse, in the app UI, the `title` property 😳 ) to an icon.